### DISCUSSION TITLE
whether reid datasets can be used or not?

### DISCUSSION THREAD

**Discussion by louyanyang** [STVIR/pysot#174] [General]
@lb1100  hi, thank you for your good job.  Recently, I was thinking about whether reid datasets could be used to enhance  discrimination of instance of the same category, but in reid datasets the space context of z and x is a problem. do you have a good idea?

**Comment by LiyaoTang** [comment]
I think that's really a good idea, but I could imagine it would bring a lot of improvement for classification branch but not so much for the detection branch since it seems the input image is already a tight crop of the target in ReID dataset and thus there is not much to regress.

correct me if I am wrong.

**Comment by ZhiyuanChen** [comment]
What do you mean by z and x?
